packages/fragno/src/mod-client.ts

```
@@ -63,10 +64,19 @@ export function instantiate(_definition: unknown) {
+    //   deps: {},
+    //   options: {},
+    //   linkedFragments: {},
+    // },
+    get $internal() {
+      throw new Error("Not supported in browser builds");
+
+      return {
+        deps: {},
+        options: {},
+        linkedFragments: {},
+      };
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)